### PR TITLE
BaSP-TG-81: Timesheets redux - Get

### PR DESCRIPTION
**Pull request by MaxiG-dev**
- Create constants to get timesheets
- Create actions to get timesheets
- Create thunk to get timesheets
- Create timesheets reducer
- Connect list component with store to get timesheets
- Add css for loading spinner

### CODE REVIEW

**Review by CristianBucci — APPROVED** [review]
Everything looks right!!

File: src/redux/timesheets/actions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import {
+  GET_TIMESHEETS_PENDING,
+  GET_TIMESHEETS_SUCCESS,
+  GET_TIMESHEETS_ERROR,
+  TOGGLE_MESSAGE_MODAL
+} from './constants';
+
+export const getTimeSheetsLoading = () => {
```
Comment (nicolasblasco) [inline src/redux/timesheets/actions.js:8]: Change action name to getTimeseetsPending

File: src/redux/timesheets/actions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  return {
+    type: GET_TIMESHEETS_PENDING
+  };
+};
+
+export const getTimeSheetsSuccess = (data) => {
```
Comment (nicolasblasco) [inline src/redux/timesheets/actions.js:14]: Change action name to getTimesheetsSuccess

File: src/redux/timesheets/actions.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  return {
+    type: GET_TIMESHEETS_SUCCESS,
+    payload: data
+  };
+};
+
+export const getTimeSheetsError = (error) => {
```
Comment (nicolasblasco) [inline src/redux/timesheets/actions.js:21]: Change action name to getTimesheetsError

File: src/redux/timesheets/reducer.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+const INITIAL_STATE = {
+  isLoading: false,
+  list: [],
+  error: '',
+  modalContent: { title: '', content: '' },
+  showModalMessage: false
+};
+
+const timeSheetsReducer = (state = INITIAL_STATE, action) => {
```
Comment (nicolasblasco) [inline src/redux/timesheets/reducer.js:16]: Rename reducer to timesheetsReducer

File: src/redux/timesheets/thunks.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { getTimeSheetsLoading, getTimeSheetsSuccess, getTimeSheetsError } from './actions';
+
+export const getTimeSheets = () => {
```
Comment (nicolasblasco) [inline src/redux/timesheets/thunks.js:3]: Rename thunk to getTimesheets

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

File: src/redux/timesheets/actions.js (hunk 1 above)
Comment (MaxiG-dev) [reply src/redux/timesheets/actions.js:8]: Done!

File: src/redux/timesheets/reducer.js (hunk 4 above)
Comment (MaxiG-dev) [reply src/redux/timesheets/reducer.js:16]: Done!

File: src/redux/timesheets/thunks.js (hunk 5 above)
Comment (MaxiG-dev) [reply src/redux/timesheets/thunks.js:3]: Done!

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**